addons/point_of_sale/views/product_view.xml

```diff
@@ -87,13 +87,13 @@
                 </page>
             </xpath>
             <xpath expr="//page[@name='purchase']" position="attributes">
-                <attribute name="invisible">detailed_type == "combo"</attribute>
+                <attribute name="invisible" separator=" or " add="detailed_type == 'combo'"></attribute>
             </xpath>
             <xpath expr="//page[@name='inventory']" position="attributes">
-                <attribute name="invisible">detailed_type == "combo"</attribute>
+                <attribute name="invisible">type in ['combo', 'service']</attribute>
             </xpath>
             <xpath expr="//page[@name='invoicing']" position="attributes">
-                <attribute name="invisible">detailed_type == "combo"</attribute>
+                <attribute name="invisible">type in ['combo', 'service']</attribute>
```

Comment:
For clean inheritance and extension, shouldn't it be an `or type == 'combo'` instead of overwriting the whole attribute ?

Reply:
`<attribute name="invisible" add="or type == 'combo'" separator=" or "/>`
instead, but yes

Reply:
@Feyensv 
Oh yes, indeed we can also do that, sorry. 🙈 

We can create a PR to change that if you prefer. 🙂